# Introduction

Proteins and Nucleic acids (DNA and RNA) are the fundamental building blocks of all life on earth. The famous “central dogma” of life, proposed by Francis Crick (Crick 1970) states that there is a sequential flow of information from nucleic acids to proteins, and not backwards. No exceptions to the crick’s definition of central dogma has been observed till date. However, research done before and after the proposal of central dogma has established that this flow of information could be regulated, either through nucleic acids themselves, or through proteins.

A “gene” is the major component of central dogma. The overall definition of a gene has been debated and revised to include protein-coding genes, regulatory elements such as “promoters” and “enhancers”, and regions that produce non-coding RNAs (Epp 1997) (Gerstein et al. 2007). A protein-coding gene transfers it’s encoded information into the protein product, through “transcription”, and “translation” (Appendix B). DNA elements that regulate this flow of information (or “expression”), such as promoters or enhancers, are stably inherited through generations and contribute to evolution (Berthelot et al. 2018). Apart from DNA elements, other factors such as semi-stable modifications to DNA elements (eg. methylation), non-coding RNAs (eg. antisense RNA (Villegas and Zaphiropoulos 2015), micro-RNA (Catalanotto et al. 2016), long non-coding RNA (Engreitz et al. 2016) etc.), proteins that interact with DNA (histone marks, transcription factors), and proteins that interact with mRNA (Siomi and Dreyfuss 1997) also regulate gene expression. Many of these gene regulatory mechanisms are established either in response to environmental changes or through genetic programming, and in some cases, could be stably maintained through cell cycle, or across multiple generations. Such mechanisms are collectively studied under the term “epigenetics” (Berger et al. 2009).

In order to understand the regulation of gene expression, it’s important to study how various genetic and epigenetic regulators interact with each other. Advances in next generation sequencing (NGS) (Shendure et al. 2005; Schuster 2008) have allowed us to study these interactions at multiple scales, ranging from local gene neighborhood to overall organisation of a chromosome. In the following sections, I would discuss the various scales of gene regulation, as well as current state of the art techniques to study them.

Multiple scales of genomic regulation
Local scale : DNA elements, histone marks and transcription factors

The “selected effect” definition of biological function (Griffiths 2009) implies that all functional elements should be under some sort of selective pressure. The primary evolutionary analysis of mammalian genome therefore proposed ~5% of human-mouse genome to be functional (Mouse Genome Sequencing Consortium et...). This estimate, which was already about 4-times larger than the protein-coding fraction (Church et al. 2009)), was further expanded in later analysis  (Rands et al. 2014; Graur 2017). Genome-wide analysis have also proposed “biochemical function” for many new genetic elements (ENCODE Project Consortium 2012). Most of these non-coding DNA elements such as promoters, enhancers, silencers, insulators and transposons, have been classically known to be involved in regulation of protein-coding genes (Riethoven 2010) (Table-1).

It has become increasingly clear that DNA elements do not act alone, but rather within the influence of the histone code (Huang et al. 2014). In fact various evidences suggest that gene regulation happens via a cross-talk between transcription factors, DNA elements, DNA modifications (such as methylation) and histone marks. For example, during embryonic development, pluripotency genes Oct3/4 are switched off in a multi-step process where first a repressor binds to it’s promoter, turning off  transcription, followed by recruitment of histone methyltransferase and histone deacetylase enzymes leading to transcriptional repression and H3K9 methylation. This follows binding of DNMT3A and DNMT3B methyltransferases, leading to promoter methylation and ultimately gene suppression (Fuhrmann et al. 2001; Feldman et al. 2006). Apart from this cross-talk, the physical distance between regulatory elements and their targets also influence gene expression. Chromatin marks such as H3K9 methylation could spread into large domains, while TFs are suggested to induce looping of chromatin in order to achieve spatial proximity to their target genes (Smallwood and Ren 2013). This looping is regulated via insulators, adding additional layer to gene regulation.





Fig 1 : Regulation of gene expression by various genetic and epigenetic elements. A promoter (1) recruits RNAP-II (2) to perform transcription of target genes. Elements such as repressors (3) or enhancers (4) could recruit transcription factors that either suppress or activate gene expression. All these elements are marked by various histone marks, such as H3K4me3 at promoters (5a) H3K36me3 at gene bodies (5b) or H3K27ac at enhancers (5c), which facilitate gene expression. Insulators (6) recruit proteins that facilitate DNA looping, and block transcription factors from activating genes outside of the established boundaries (7).
