`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'`


# Publications and Manuscripts

## Analysis of chromosome conformation in flies

+------------------------------------------------------------------------------------------+
|  **High-resolution TADs reveal DNA sequences underlying genome  organization in flies.** |
|  Ramírez, F.\*, **Bhardwaj, V.\***, Arrigoni,                                            |
|  L., Lam, K. C., Grüning, B. A., Villaveces, J., ... Manke, T.                           |
|  **Nature communications (2018)**. doi:10.1038/s41467-017-02525-w                        |
|  \* shared authorship                                                                    |
+------------------------------------------------------------------------------------------+


I contributed to the development of HiCExplorer and HiCBrowser (led by Fidel Ramirez), and developed the Chorogenome Navigator resource. I performed the analysis of motif combinations and boundary strength, prediction of motifs, and relationship of TAD boundaries and transcription (Fig 2, S2, 4, S4, 5 and S5). Together with Fidel Ramirez and Thomas Manke I devised, wrote, and revised the manuscript.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/NatComm_2018_article.pdf}

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/NatComm_2018_supple.pdf}

## Galaxy HiCExplorer

+--------------------------------------------------------------------------------------------------------------+
| **Galaxy HiCExplorer: a web server for reproducible HiC data analysis, quality control and visualization.** |
| Wolff, J., **Bhardwaj, V.**, Nothjunge, S., Richard, G., Renschler, G., Gilsbach, R., ... Grüning, B.A.      |
| **Nucleic acids research (2018).** doi:10.1093/nar/gky504                                                |
+--------------------------------------------------------------------------------------------------------------+

I contributed to the development of HiCExplorer and designed the template for the workflow used in the galaxy web server. I contributed to the writing and revision of the manuscript along with Joachim Wolff and other authors.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/NAR_2018_article.pdf}

## Analysis of dosage compensation in flies via promoter-profiling

+------------------------------------------------------------------------------------+
| **Quantitative analysis of dosage compensation in flies using promoter profiling** |
| Bhardwaj V.\*, Semplicio G\*, Manke T, Akhtar A *(unpublished)*.                   |
| \* shared authorship                                                               |
+------------------------------------------------------------------------------------+


I contributed to the development of the MAPCap protocol by providing the analysis input, where all the experiments were performed by Giuseppe Semplicio. I developed the icetea bioconductor package, performed all the analysis presented in the paper, made the figures and wrote the manuscript with input from Giuseppe Semplicio.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/mapcap_paper_14-09-2018.pdf}


## Interaction of MLE ortholog DHX9 with Alu elements in the human genome


+-------------------------------------------------------------------------------------------------------------+
| **DHX9 suppresses RNA processing defects originating from the Alu invasion of the human genome.**           |
| Aktaş, T., Avşar Ilık, İ., Maticzka, D., **Bhardwaj, V.**, Pessoa Rodrigues, C., Mittler, G., ... Akhtar, A.|
| **Nature (2017)**. doi:10.1038/nature21715                                                                  |
+-------------------------------------------------------------------------------------------------------------+


I performed the analysis of UV-CLAP data for estimation of Alu enrichment (Extended data Fig. 2 and 3) and performed the RNA-seq analysis for detection of differential expression, splicing, circular RNAs and RNA editing (Fig. 2a, 2b, 3a, Extended Data Figure 6, 7, 10). I contributed to the writing and revision of the manuscript along with Tugce Aktas, Ibrahim Ilik, Asifa Akhtar and other authors.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/Nature_2017_article_supple.pdf}


## Update of the deepTools toolkit for exploring deep-sequencing data

+-----------------------------------------------------------------------------------------------------+
| **deepTools2: a next generation web server for deep-sequencing data analysis.**                     |
| Ramírez, F., Ryan, D. P., Grüning, B., **Bhardwaj, V.**, Kilpert, F., Richter, A. S., ... Manke, T. |
| **Nucleic acids research (2016)**. doi:10.1093/nar/gkw257                                           |
+-----------------------------------------------------------------------------------------------------+


I contributed to the development of deepTools2 (led by Fidel Ramirez and Devon Ryan) through features and bugfixes, and to the testing and update of its documentation and the galaxy server. I helped with the writing and revision of the manuscript by Fidel Ramirez and other authors.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/NAR_2016_article.pdf}


## snakePipes enables reproducible epigenomic analysis


+--------------------------------------------------------------------------------------------------+
| **snakePipes enable flexible, scalable and integrative epigenomic analysis.**                    |
| **Bhardwaj, V.\***, Heyne, S.\*, Sikora, K., Rabbani, L., Rauer, M., Kilpert, F., ... Manke, T.  |
| **bioRxiv (2018).** doi:10.1101/407312                                                           |
| \* shared authorship                                                                             |
+--------------------------------------------------------------------------------------------------+


I developed the allele-specific and HiC workflows and contributed to DNA-mapping, ChIP-seq, ATAC-seq and RNA-seq workflows and documentation. I performed the analysis, prepared the figures and wrote the manuscript with input from all authors.

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm]{manuscripts/snakepipes_manuscript_10-09-2018.pdf}

<!--------- Tables and definitions ----->

# Supplemental information

\clearpage

## Definitions

**DNA :** Deoxyribonucleic acid. It is a negatively charged, large
polymer composed of a combination of four bases (Adenine, Guanine,
Cytosine, Thymine) and stores genetic information in living organisms.

**Histones :** Negatively charged proteins that wrap the DNA in the
nucleus. Apart from providing structural support, the histones are also
critical in gene regulation.

**Chromatin :** In order to efficiently pack inside the nucleus, DNA is
wrapped around positively charged proteins called "histones" in order to
form "chromatin".

**Transcription :** production of messenger RNA or "mRNA" through DNA,
by the protein "RNA polymerase"

**Translation :** production of proteins through mRNA by RNA-protein
complex "Ribosome"

**Histone Code :** Numerous post-translational modifications are
possible on histone marks, usually referred to as histone code ^19^.

**Readers** : Proteins that recognize histone marks ^206^. Readers
either perform various biochemical activities in response to the histone
code, or interact with other proteins that perform these activities
eventually leading to the regulation of target genes ^207,208^.

**Writers :** The proteins that deposit histone marks either in response
to change in cellular environment, through genetic programming (eg.
during development), or in response to another histone mark ^209^.

**Transcription factors (TFs) :** Classically, TFs are described as
having a DNA binding domain, and a trans-activation domain, which
activates RNAP-II machinery to facilitate transcription ^210^. TFs
could also recruit various co-activators or co-repressors which add
additional complexity to transcriptional regulation ^211^.

**General TFs :** Usually, transcription factors associated with core
RNAP-II machinery are referred to as "general transcription factors"
(GTFs) [^212^] while the specific transcription factors associated with
enhancers and RNAP-II under specific conditions, are referred simply as
"transcription factors" (TFs) ^213^.

**DNA motifs :** Short DNA sequences that are recognized by DNA binding
domains of various regulatory proteins.

**4-cutters :** Restriction enzymes with 4 base-pair recognition sites on DNA. Eg. Dpn-II

**6-cutters :** Restriction enzymes with 6 base-pair recognition sites on DNA. Eg. Hind-III

## Tables

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm,angle=90]{data/Table-1.pdf}

\includepdf[scale=0.9,pages=-,pagecommand={}, offset=0.3cm 0cm,angle=90]{data/Table-2.pdf}

<!--------- CV ----->

# Academic Vita
\setstretch{1.2}

#### EDUCATION

**Since October 2014 : PhD Candidate**

Max Planck Institute of Immunobiology and Epigenetics and Faculty of Biology, University of Freiburg

**2011 - 2013 : Master of Science (Biomedical Sciences)**

University of Delhi, New Delhi, India

Thesis : Exploring the novel regulatory mechanisms of gene regulation by
Human Alu elements.

**2008 - 2011 : Bachelor of Science (Biomedical Sciences)**

University of Delhi, New Delhi, India

Thesis: Gene Ontology Annotation of Mycobacterium tuberculosis H37Rv
genes.

**2008 : High School Diploma**

Central Board of Secondary Education, India

#### PUBLICATIONS/PREPRINTS

**snakePipes enable flexible, scalable and integrative epigenomic
analysis.** **Bhardwaj, V.\***, Heyne, S.\*, Sikora, K., Rabbani, L.,
Rauer, M., Kilpert, F., ... Manke, T. **bioRxiv (2018).**
[[https://doi.org/10.1101/407312]{.underline}](https://doi.org/10.1101/407312)

**High-resolution TADs reveal DNA sequences underlying genome
organization in flies.** Ramírez, F.\*, **Bhardwaj, V.\***, Arrigoni,
L., Lam, K. C., Grüning, B. A., Villaveces, J., ... Manke, T. **Nature
communications (2018)**.
[[doi:10.1038/s41467-017-02525-w]{.underline}](https://www.nature.com/articles/s41467-017-02525-w)

**Galaxy HiCExplorer: a web server for reproducible Hi-C data analysis,
quality control and visualization.** Wolff, J., **Bhardwaj, V.**,
Nothjunge, S., Richard, G., Renschler, G., Gilsbach, R., ... Grüning, B.
A. **Nucleic acids research (2018).**
[[doi.org/10.1093/nar/gky504]{.underline}](http://doi.org/10.1093/nar/gky504)

**Community-Driven Data Analysis Training for Biology.** Batut, B.,
Hiltemann, S., Bagnacani, A., Baker, D., **Bhardwaj, V.**, Blank, C.,
... Grüning, B. **Cell Systems* (2018).**
[[doi:10.1016/j.cels.2018.05.012]{.underline}](https://doi.org/10.1016/j.cels.2018.05.012)

**DHX9 suppresses RNA processing defects originating from the Alu
invasion of the human genome.** Aktaş, T., Avşar Ilık, İ., Maticzka, D.,
**Bhardwaj, V.**, Pessoa Rodrigues, C., Mittler, G., ... Akhtar, A.
**Nature (2017)**.
[[doi:10.1038/nature21715]{.underline}](https://www.nature.com/articles/nature21715)

**deepTools2: a next generation web server for deep-sequencing data
analysis.** Ramírez, F., Ryan, D. P., Grüning, B., **Bhardwaj, V.**,
Kilpert, F., Richter, A. S., ... Manke, T. **Nucleic acids research
(2016)**.
[[doi:10.1093/nar/gkw257]{.underline}](http://dx.doi.org/10.1093/nar/gku365)

**MOF maintains transcriptional programs regulating cellular stress
response.** Sheikh, B. N., Bechtel-Walz, W., Lucci, J., Karpiuk, O.,
Hild, I., Hartleben, B., **\*\***... Akhtar, A. **Oncogene* (2016)**.
[[doi:10.1038/onc.2015.335]{.underline}](https://doi.org/10.1038/onc.2015.335)

**Alu-miRNA interactions modulate transcript isoform diversity in stress
response and reveal signatures of positive selection.** Pandey, R.,
Bhattacharya, A., **Bhardwaj, V.**, Jha, V., Mandal, A. K., & Mukerji,
M. **Scientific Reports* (2016)**.
[[doi:10.1038/srep32348]{.underline}](http://dx.doi.org/10.1038/srep32348)

**Draft Genome Sequence of Urease-Producing Sporosarcina pasteurii with
Potential Application in Biocement Production.** Tiwari, P. K.\*, Joshi,
K.\*, Rehman, R.\*, **Bhardwaj, V.\***, Shamsudheen, K. V., Sivasubbu,
S., & Scaria, V. **Genome Announcements* (2014)**.
[[doi:10.1128/genomeA.01256-13]{.underline}](https://doi.org/10.1128/genomeA.01256-13)

**The Zebrafish GenomeWiki: a crowdsourcing approach to connect the long
tail for zebrafish gene annotation.** Singh, M., Bhartiya, D., Maini,
J., Sharma, M., Singh, A. R., Kadarkaraisamy, S., **\*\*\***...
Sivasubbu, S. **Database: The Journal of Biological Databases and
Curation* (2014)**
[[doi:10.1093/database/bau011]{.underline}](https://doi.org/10.1093/database/bau011)

\* contributed equally

\*\* author at 10th position

\*\*\* author at 27th position
